packages/select/index.js

```diff
@@ -120,7 +127,8 @@ export class WarpSelect extends kebabCaseAttributes(WarpElement) {
           aria-describedby="${ifDefined(this.#helpId)}"
           aria-invalid="${ifDefined(this.invalid)}"
           aria-errormessage="${ifDefined(this.invalid && this.#helpId)}"
-          @keydown=${this.handleKeyDown}>
+          @keydown=${this.handleKeyDown}
+          @change=${(e) => this.onChange(e.target.value)}>
```

Comment:
Maybe cleaner if you just provide the function here as a callback `@change=${this.onChange}` and then set the value in the function as `{ detail: e.target.value }`

Reply:
Nice! Updated it.